app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -267,7 +267,7 @@ def system_purpose
     api :POST, "/hosts/bulk/available_incremental_updates", N_("Given a set of hosts and errata, lists the content view versions" \
                                                                  " and environments that need updating.")
     param_group :bulk_params
-    param :errata_ids, Array, :desc => N_("List of Errata ids")
+    param :errata_ids, Array, :desc => N_("List of Errata ids"), :required => true
```

Comment:
Does the API give an error if the errata_ids arent specified? Might be a good addition if not

Reply:
@jturel updated and included a test to confirm we get an error when not sending the param :)